skimage/viewer/plugins/color_histogram.py

```diff
@@ -27,7 +27,7 @@ def _on_new_image(self, image):
         # Calculate color histogram in the Lab colorspace:
         L, a, b = self.lab_image.T
         left, right = -100, 100
-        ab_extents = [left, right, right, left]
+        ab_extents = [left - 0.5, right + 0.5, right + 0.5, left - 0.5]
```

Comment:
This change seems unrelated to this PR. Please revert. (And this module is deprecated, will be removed soon, so no need to maintain this code).